activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Allow destroying active storage variants
+
+    ```ruby
+    attachable.variant(resize_to_limit: [100, 100]).destroy
```

Comment:
```suggestion
    User.avatar.variant(resize_to_limit: [100, 100]).destroy
```

Reply:
Mmm, `avatar` is not a class method. So how about `User.first.avatar...`

Reply:
Fixed it. Thank you for the feedback :+1: